src/axom/core/docs/sphinx/core_containers.rst

```diff
@@ -137,6 +145,72 @@ The output of this example is::
   Range-based for loop over ArrayView c yields: 1 5 6 9 1 4
   Standard for loop over ArrayView c yields: 1 5 6 9 1 4
 
+########################
+Using Arrays in GPU Code
+########################
+
+Instead of writing kernels and device functions that operate on raw pointers, we can use ``ArrayView``
+in device code. The basic "workflow" for this process is as follows:
+
+  1. Create an ``Array`` allocated in device-accessible memory via either specifying an allocator ID
+     or using a class template parameter for the desired memory space.
+  2. Write a kernel that accepts an ``ArrayView`` parameter **by value**, not by reference or pointer.
+  3. Create an ``ArrayView`` from the ``Array`` to call the function.  For non-templated kernels
+     an implicit conversion is provided.
```

Comment:
>   Create an ``ArrayView`` from the ``Array`` to call the function.

Could we please spell this out a bit more explicitly in the discussion below?

One option here is to use the following `ArrayView` constructor -- 
```cpp
Array<int, 1, DEVICE> arr(n);
ArrayView(arr);
```

Another option is the implicit conversion that was mentioned.

Are there other ways?

Reply:
I've added a new section to the example + docs that gets into the explicit + implicit conversions to ArrayView before diving into the `add` kernel.